InAppPurchasesAtlasAppServices/FoodieFolio/FoodieFolio/Views/RecipesListView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  RecipesListView.swift
+//  FoodieFolio
+//
+//  Created by Mar Cabrera on 09/08/2023.
+//
+
+import SwiftUI
+import RealmSwift
+
+struct RecipesListView: View {
+    @ObservedResults(Recipe.self) var recipes: Results<Recipe>
+    @ObservedResults(Purchase.self) var purchases: Results<Purchase>
+    @State private var isPresentingModal = false
+    @State private var selectedCategory: String = "general"
+
+    var body: some View {
+        TabView {
+            NavigationStack {
+                VStack(alignment: .leading) {
+                    VStack(alignment: .leading) {
+                        Text("Category")
+                            .font(.system(size: 20, weight: .semibold))
+                        HStack {
+                            CapsuleButton(title: "General") {
+                                selectedCategory = "general"
+                            }
+                            if showPremiumButton {
+                                CapsuleButton(title: "Premium") {
+                                    selectedCategory = "premium"
+                                }
+                            }
+                        }
+                        Divider()
+
+                    }.padding()
+
+                    List(filteredRecipes, id: \._id) { recipe in
+                        NavigationLink {
+                            RecipeDetail(recipe: recipe)
+                        } label: {
+                            RecipeRow(recipe: recipe)
+                        }
+                    }
+                    .listStyle(.plain)
+                    .navigationBarTitle("Recipes")
+                    .navigationBarItems(trailing: Button {
+                        isPresentingModal.toggle()
+                    } label: {
+                        Image(systemName: "gift.fill")
+                            .foregroundColor(ColorPalette.lightLilac)
+                    })
+                    .sheet(isPresented: $isPresentingModal) {
+                        StorePurchaseView()
+                    }
+                }
```

Comment:
Changing the `navigation` modifiers to the VStack instead of attaching it to the List, will help with readability

Reply:
I was thinking the outer VStack